src/plugins/core/borders.ts

```diff
@@ -219,6 +219,21 @@ export class BordersPlugin extends CorePlugin<BordersPluginState> implements Bor
     return Object.keys(sheetBorders).map((index) => parseInt(index, 10));
   }
 
+  /**
+   * Get all the rows which contains at least a border
+   */
+  private getRowsWithBorders(sheetId: UID): number[] {
+    const sheetBorders = this.borders[sheetId]?.filter(isDefined);
+    if (!sheetBorders) return [];
+    const rowsWithBorders = new Set<number>();
+    for (const border of sheetBorders) {
+      for (const rowBorder of Object.keys(border)) {
```

Comment:
```suggestion
      for (const rowBorder in border) {
```
small nitpick,i'll change it myself, but you can just iterate on the index of an array with `(element **in** array)`